src/windows/export.py

```diff
@@ -136,12 +136,12 @@ def __init__(self):
         self.timeline.Open()
 
         # Default export path
-        recommended_path = recommended_path = os.path.join(info.HOME_PATH)
+        recommended_path = os.path.join(info.HOME_PATH)
```

Comment:
This has actually been like that since 2016! None of us noticed. :laughing:

Reply:
@ferdnyc - Oh dear. This is funny. 😆